helpers/pathContentTypeSupported.js

```diff
@@ -2,21 +2,19 @@ const supportedTypes = ["application/json", "text/plain"];
 const arrayCount = supportedTypes.length - 1;
 
 const pathContentTypeSupported = (path) => {
-  for (let i = 0; i <= arrayCount; i++) {
-    if (path.requestBody) {
-      if (path.requestBody.content) {
+  if (path.requestBody) {
+    if (path.requestBody.content) {
```

Comment:
I don't think `if (path.requestBody)` is required here. The second if statement should suffice